src/components/ProgressBar/ProgressBar.module.css

```diff
@@ -0,0 +1,31 @@
+@keyframes indeterminate {
+  0% {
+    transform: translateX(-100%);
+    width: 100%;
+  }
+  100% {
+    transform: translateX(200%);
+    width: 50%;
+  }
+}
+
+.progressBarIndeterminate {
+  animation: indeterminate 1.5s ease-in-out 0s infinite none running;
+}
+
+.progressBarIndicator {
+  position: relative;
+  display: block;
+  height: 100%;
+  transition: transform 0.2s linear;
+  background: green;
+}
+
+.progressBarTrack {
+  border-radius: 2px;
+  overflow: hidden;
+  height: 12px;
+  position: relative;
+  z-index: -1;
```

Comment:
Any potential issues with a `-1` zIndex here, given the parent's position isn't specified? Why not `0` here and a `1` for the indicator? Just wondering if the track might disappear if parent elements have a defined positive `zIndex` or something.

Reply:
Ah, yeah, that was leftover from a previous styling attempt when using absolute positioning. Good eye!